apps/protocol-frontend/src/components/HomeShell.tsx

```diff
@@ -1,13 +1,22 @@
 import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { useWallet } from '@raidguild/quiver';
-import { Flex, Link as ChakraLink, Button, Text, Box } from '@chakra-ui/react';
+import {
+  Flex,
+  Link as ChakraLink,
+  Button,
+  Text,
+  Box,
+  Icon,
+} from '@chakra-ui/react';
 import ConnectWallet from '../components/ConnectWallet';
 import PageHeading from './PageHeading';
 import CreateUserForm from './CreateUserForm';
 
 import { useUser } from '../contexts/UserContext';
 import CreateWaitlistUserForm from './CreateWaitlistUserForm';
+import { GOVRN_MOTTO } from '../utils/constants';
```

Comment:
Nice. I have a few other things we can add to this now that it is here (constants.ts)